pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -59,7 +48,8 @@ in mkDerivation rec {
     qtbase qtimageformats gtk3 libsForQt5.kwayland libsForQt5.libdbusmenu enchant2 lz4 xxHash
     dee ffmpeg openalSoft minizip libopus alsa-lib libpulseaudio range-v3
     tl-expected hunspell glibmm webkitgtk
-    tg_owt libtgvoip
+    libtgvoip rnnoise abseil-cpp extra-cmake-modules
```

Comment:
It's weird if that worked previously, libtgvoip's bundled webrtc should conflict with tg_owt in one address space

Reply:
With Telegram-Desktop nothing's weird :o xD (SCNR)

Anyway, not sure if it worked previously, I've never used that functionality myself (in fact I luckily don't really have to use Telegram anymore so I'll probably remove myself as maintainer soon - I'm currently just maintaining it because I haven't found a replacement yet but this update required much more effort than expected...).

Thanks a lot for the information though :) Hopefully someone how uses that functionality can have a look at it. This might explain #126285.

Reply:
I forgot that `tg_owt` links statically. That should avoid the address space collision then (at least AFAIK).

Reply:
libtgvoip's webrtc has another, older API, are you sure it can't modify the same structures when it's calling this API?

Reply:
Oh, I thought you referred to the shared libraries diamond dependency problem. I have no idea if (and how) some data structures are shared between `tg_owt` and `libtgvoip` (so no idea if that causes issues or not).